src/pkg/utils/io.go

```diff
@@ -249,7 +249,11 @@ func CreatePathAndCopy(source string, destination string) error {
 	}
 
 	// Copy all the source data into the destination location
-	if err := copy.Copy(source, destination); err != nil {
+	if err := copy.Copy(source, destination, copy.Options{
+		OnSymlink: func(_ string) copy.SymlinkAction {
+			return copy.Deep
+		},
+	}); err != nil {
```

Comment:
Did you change this back?

Reply:
It shouldn't enter into that since a symlink isn't a regular file and is header-only in a tarball

Reply:
I'm working on a change now.

Reply:
You will run into this on a deploy, but that is more to do with what Zarf expects manifests are (the test would likely need to be refactored into a local chart)
![image](https://github.com/defenseunicorns/zarf/assets/3977569/c8ebbdd4-9b7e-4499-86a6-175c36d810f0)